lib/core/src/server/preview/iframe-webpack.config.js

```diff
@@ -70,8 +70,19 @@ export default async ({
 
       virtualModuleMapping[entryFilename] = entryTemplate
         .replace('{{configFilename}}', configFilename.replace(/\\/g, '/'))
-        .replace('{{clientApi}}', clientApi.replace(/\\/g, '/'))
-        .replace('{{clientLogger}}', clientLogger.replace(/\\/g, '/'));
+        .replace(
```

Comment:
Can you make this DRY?

Reply:
Something like:

```
.replace('{{clientApi}}', _escape(clientApi))
.replace('{{clientLogger}}', _escape(clientLogger))
```

Reply:
I added a commit to extract an interpolation function

Reply:
nice!